pkg/securitycontext/util_test.go

```diff
@@ -84,6 +84,10 @@ func compareContexts(name string, ex, ac *api.SELinuxOptions, t *testing.T) {
 	}
 }
 
+func container(*int64) api.Container {
```

Comment:
nit: I would make this `containerWithUser` and give a name to the method parameter for clarity.